crates/iota-core/src/unit_tests/authority_aggregator_tests.rs

```diff
@@ -964,7 +964,7 @@ async fn test_handle_transaction_response() {
     let resp = HandleTransactionResponse {
         status: TransactionStatus::Executed(
             Some(cert_epoch_0.auth_sig().clone()),
-            sign_tx_effects(effects, 0, *name_0, key_0),
+            Box::new(sign_tx_effects(effects, 0, *name_0, key_0)),
```

Comment:
Does this exist in the upstream change PR?

Reply:
Same here:
No, but without it, it introduced a cargo check / CI warning on our side (with our CI checks settings)
Moving the enum variant into a Box removes that warning.

Reply:
and same thing in a couple other places where that change was ported. 
Is this not the right solution ? and if no, why ?

Reply:
-> removed the box in the end since the problem it solves is going away in an ucoming commit port